pkg/common/types_shootflavor.go

```diff
@@ -63,6 +66,10 @@ type ShootFlavor struct {
 	// Kubernetes versions to test
 	KubernetesVersions ShootKubernetesVersionFlavor `json:"kubernetes"`
 
+	// AllowPrivilegedContainers defines whether privileged containers will be allowed in the given shoot or not
+	// +optional
+	AllowPrivilegedContainers string `json:"allowPrivilegedContainers"`
```

Comment:
why are you not using a boolean.
if you need to distinguish between tru, false and not set I would suggest to use a boolean pointer?

A string seams strange as you would always require quotes.

Reply:
Although helm values handling on boolean is a bit annoying (e.g. https://github.com/helm/helm/issues/3308) I switched to bool pointers.